### PR TITLE
chore(docker): set up docker + ci

### PR DESCRIPTION
**Pull request by guillaume-chervet**
for closing #1

### CODE REVIEW

File: .github/workflows/dailyclean-api-docker-image.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+name: DailyClean API Docker Image CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+#   paths:
```
Comment (guillaume-thomas) [inline .github/workflows/dailyclean-api-docker-image.yml:8]: suggestion: Remove these comments ?

File: .github/workflows/dailyclean-job-docker-image.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+name: DailyClean JOB Docker Image CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+   # paths:
```
Comment (guillaume-thomas) [inline .github/workflows/dailyclean-job-docker-image.yml:8]: suggestion: Remove these comments ?

File: api/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    
+FROM registry.access.redhat.com/ubi8/ubi:8.5-200 AS build
+
+ENV APP_ROOT /app_root
+ENV HOME /home
+
+WORKDIR ${APP_ROOT}
+USER root
+
+ENV MAVEN_VERSION 3.6.3
```
Comment (guillaume-thomas) [inline api/Dockerfile:21]: suggestion: old version of maven. Is it possible to bump to 3.8.X ?

**Review by guillaume-thomas — APPROVED** [review]
Somes little ajustments are possible, but this PR is gold <3

File: api/Dockerfile (hunk 3 above)
Comment (guillaume-chervet) [reply api/Dockerfile]: Done

File: .github/workflows/dailyclean-job-docker-image.yml (hunk 2 above)
Comment (guillaume-chervet) [reply .github/workflows/dailyclean-job-docker-image.yml]: Done

File: .github/workflows/dailyclean-api-docker-image.yml (hunk 1 above)
Comment (guillaume-chervet) [reply .github/workflows/dailyclean-api-docker-image.yml]: Done

**Pull request merged**